source/opt/CMakeLists.txt

```diff
@@ -88,6 +89,7 @@ add_library(SPIRV-Tools-opt
   local_single_block_elim_pass.cpp
   local_single_store_elim_pass.cpp
   local_ssa_elim_pass.cpp
+  merge_return_pass.cpp
```

Comment:
Need a corresponding change to Android.mk, to support Android builds via ndk-build.

Reply:
Thanks, fixed.